components/src/Tables/Table.js

```diff
@@ -33,31 +82,13 @@ export const ActionCell = styled(Cell)`
     white-space: nowrap;
     vertical-align: middle;
     & > button:not(:last-child) {
-      margin-right: ${theme.space.half}px;
```

Comment:
Have you considered the usbalitly of adding a large non-interactive space between buttons? I often find it easier to use button-groups where the buttons themselves have a large target zone, while the space between buttons is small.

Reply:
I have and I agree with that. In this case, buttons are squeezed a bit more to save some space and make "responsive demo" more attractive. Having said that I will add extra space now that Shah demo is over.